package.json

```diff
@@ -19,6 +21,7 @@
   },
   "scripts": {
     "start": "react-scripts start",
+    "server": "nodemon ./server/index.js",
     "build": "react-scripts build",
     "test": "jest --watch",
     "eject": "react-scripts eject"
```

Comment:
I'm not sure why it's not reflected on the main branch (I probably haven't pushed this change) but the README uses these scripts:

```
"scripts": {
    "dev": "react-scripts start",
    "start": "nodemon ./server/index.js",
    "build": "react-scripts build",
    "test": "jest --watch",
    "eject": "react-scripts eject"
  }
  ```
  
  It'd be awesome if you could just remove the server script, and move its command up to start. You could also just copy the block above over.
  
  Thank you!